### PR TITLE
add cookbook for nn regression

### PR DESCRIPTION
**Pull request by sanuj**

### CODE REVIEW

File: doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+==================================
+Feedforward Network for Regression
```
Comment (karlnapf) [inline doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst]: can you pls send a separate patch that adds the nn examples under regression and classification in the meta examples as well (so they appear multiple times), also for the gp examples.

File: doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+==================================
+
+This page illustrates the usage of feedforward networks for regression. For more details about feedforward networks, `see this feedforward network cookbook <http://shogun.ml/cookbook/latest/examples/neural_nets/feedforward_net_classification.html>`_.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst]: pls use the "doc" style of referencing.
Also pls do that for all other cookbooks you did (see my email, separate patch)

File: doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+We create a :sgclass:`CNeuralNetwork` instance by using the above layers and randomly initialize the network parameters by sampling from a gaussian distribution.
+
+.. sgexample:: feedforward_net_regression.sg:create_instance
+
+Before training, we need to set appropriate parameters like regularization coefficient, number of epochs, learning rate, etc. as shown below. More parameters can be found in the documentation of :sgclass:`CNeuralNetwork`.
+
+.. sgexample:: feedforward_net_regression.sg:set_parameters
+
+We train the model and apply it to some test data.
```
Comment (karlnapf) [inline doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst]: remove "some"

**Comment by karlnapf** [comment]
some minor adjustments and then merge ready

**Comment by sanuj** [comment]
Updated.
